.env

```
@@ -0,0 +1,3 @@
+EXPO_PUBLIC_API_KEY=3iYMnQFDLe5CFndg9pduJx2x5qFcBSNLcBBnhg8vm6zzWEUfGVgE6DelEz5k4oIRj7p707cSAjuzw7DeS5DPdZJtoWvbrnQET01vOkh9GjN3SOXAHpw4l7mqqUHUh9c0
+EXPO_PUBLIC_API_URL=https://frecap-integ.ovh/vakeo/api
+SENTRY_AUTH_TOKEN=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
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)